v2/blockstore/readonly_test.go

```diff
@@ -235,3 +235,24 @@ func newV1Reader(r io.Reader, zeroLenSectionAsEOF bool) (*carv1.CarReader, error
 	}
 	return carv1.NewCarReader(r)
 }
+
+func TestReadOnlyErrorAfterClose(t *testing.T) {
+	bs, err := OpenReadOnly("../testdata/sample-v1.car")
+	require.NoError(t, err)
+
+	roots, err := bs.Roots()
+	require.NoError(t, err)
+	_, err = bs.Get(roots[0])
+	require.NoError(t, err)
+	_, err = bs.GetSize(roots[0])
+	require.NoError(t, err)
```

Comment:
no error assertion is probably an overkill since we have tests already that assert readonly with that file behaves.

Reply:
maybe overkill, I just wanted to show the mirror image of "does not error" with "always errors" before and after closing.